test_runner/fixtures/neon_fixtures.py

```diff
@@ -1723,6 +1725,42 @@ def __init__(self, env: NeonEnv, port: PageserverPort, config_override: Optional
         self.config_override = config_override
         self.version = env.get_pageserver_version()
 
+        # After a test finishes, we will scrape the log to see if there are any
+        # unexpected error messages. If your test expects an error, add it to
+        # 'allowed_errors' in the test with something like:
+        #
+        # env.pageserver.allowed_errors.append(".*could not open garage door.*")
+        #
+        # The entries in the list are regular experessions.
+        self.allowed_errors = [
```

Comment:
It looks like a constant value and can be precompiled regexp. I'd recommend putting these errors as a class attribute and using `re.compile` for each row.